airflow/executors/base_executor.py

```diff
@@ -144,23 +156,28 @@ def trigger_tasks(self, open_slots):
             key=lambda x: x[1][1],
             reverse=True)
         for _ in range(min((open_slots, len(self.queued_tasks)))):
-            key, (command, _, queue, simple_ti) = sorted_queue.pop(0)
+            key, (command_to_run, _, _, simple_ti) = sorted_queue.pop(0)
             self.queued_tasks.pop(key)
-            self.running[key] = command
+            self.running.add(key)
             self.execute_async(key=key,
-                               command=command,
-                               queue=queue,
+                               command_to_run=command_to_run,
```

Comment:
Is `queue` not needed

Reply:
Not really. It was not used in any of the executors, so I removed it (we can always add it when needed).